databricks/koalas/tests/test_stats.py

```diff
@@ -26,149 +24,150 @@
-            self.assert_eq(getattr(ddf, funcname)(), getattr(df, funcname)())
+            self.assertEqual(getattr(kdf.A, funcname)(), getattr(pdf.A, funcname)())
+            self.assert_eq(getattr(kdf, funcname)(), getattr(pdf, funcname)())
 
         functions = ['std', 'var']
         for funcname in functions:
-            self.assertAlmostEqual(getattr(ddf.A, funcname)(), getattr(df.A, funcname)())
-            self.assertPandasAlmostEqual(getattr(ddf, funcname)(), getattr(df, funcname)())
+            self.assertAlmostEqual(getattr(kdf.A, funcname)(), getattr(pdf.A, funcname)())
+            self.assertPandasAlmostEqual(getattr(kdf, funcname)(), getattr(pdf, funcname)())
 
         # NOTE: To test skew and kurt, just make sure they run.
         #       The numbers are different in spark and pandas.
         functions = ['skew', 'kurt']
         for funcname in functions:
-            getattr(ddf.A, funcname)()
-            getattr(ddf, funcname)()
+            getattr(kdf.A, funcname)()
+            getattr(kdf, funcname)()
 
     def test_abs(self):
-        df = pd.DataFrame({'A': [1, -2, 3, -4, 5],
-                           'B': [1., -2, 3, -4, 5],
-                           'C': [-6., -7, -8, -9, 10],
-                           'D': ['a', 'b', 'c', 'd', 'e']})
-        ddf = koalas.from_pandas(df)
-        self.assert_eq(ddf.A.abs(), df.A.abs())
-        self.assert_eq(ddf.B.abs(), df.B.abs())
-        self.assert_eq(ddf[['B', 'C']].abs(), df[['B', 'C']].abs())
-        # self.assert_eq(ddf.select('A', 'B').abs(), df[['A', 'B']].abs())
+        pdf = pd.DataFrame({'A': [1, -2, 3, -4, 5],
+                            'B': [1., -2, 3, -4, 5],
+                            'C': [-6., -7, -8, -9, 10],
+                            'D': ['a', 'b', 'c', 'd', 'e']})
+        kdf = koalas.from_pandas(pdf)
+        self.assert_eq(kdf.A.abs(), pdf.A.abs())
+        self.assert_eq(kdf.B.abs(), pdf.B.abs())
+        self.assert_eq(kdf[['B', 'C']].abs(), pdf[['B', 'C']].abs())
+        # self.assert_eq(kdf.select('A', 'B').abs(), pdf[['A', 'B']].abs())
 
     def test_corr(self):
         # Disable arrow execution since corr() is using UDT internally which is not supported.
         with self.sql_conf({'spark.sql.execution.arrow.enabled': False}):
             # DataFrame
             # we do not handle NaNs for now
-            df = pd.util.testing.makeMissingDataframe(0.3, 42).fillna(0)
-            ddf = koalas.from_pandas(df)
+            pdf = pd.util.testing.makeMissingDataframe(0.3, 42).fillna(0)
+            kdf = koalas.from_pandas(pdf)
 
-            res = ddf.corr()
-            sol = df.corr()
+            res = kdf.corr()
+            sol = pdf.corr()
             self.assertPandasAlmostEqual(res, sol)
 
             # Series
-            a = df.A
-            b = df.B
-            da = ddf.A
-            db = ddf.B
+            a = pdf.A
+            b = pdf.B
+            da = kdf.A
```

Comment:
You could also rename `da` and `db` to `ka` and `kb` since adding the `k` suffix for the Koalas version of pandas variables is what you commonly see in this project.